MAUI/Calendar/getting-started.md

```diff
@@ -156,4 +156,44 @@ this.Calendar.SelectionMode = CalendarSelectionMode.Multiple;
 {% endhighlight %}  
 {% endtabs %}
 
-![change-selection-mode](images/getting-started/maui-calendar-multiple-selection.png)
\ No newline at end of file
+![change-selection-mode](images/getting-started/maui-calendar-multiple-selection.png)
+
+## Action buttons
+
+You can display action buttons at the bottom of the calendar by using the `ShowActionButtons` property of the SfCalendar. It allows you to confirm or cancel the selected date, dates, and range of dates in calendar views of the `SfCalendar`.
+
+{% tabs %}  
+{% highlight xaml tabtitle="XAML" %}
+
+<calendar:SfCalendar x:Name="Calendar" 
+                     View="Month" 
+                     ShowActionButtons="True">
+ </calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" %}
+
+this.Calendar.ShowActionButtons = true;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Today button
+
+The today button can be displayed at the bottom of the calendar using the `ShowTodayButton` property of the `SfCalendar`, allowing you to quickly navigate from current view to the today view.
```

Comment:
The today button can be displayed at the bottom of the calendar using the `ShowTodayButton` property of the `SfCalendar,` allowing you to quickly navigate from the current view to the today view.

Reply:
Content modified.